.gitmodules

```diff
@@ -16,8 +16,8 @@
   branch = develop
 [submodule "CMEPS"]
   path = CMEPS-interface/CMEPS
-  url = https://github.com/NOAA-EMC/CMEPS
-  branch = emc/develop
+  url = https://github.com/AnningCheng-NOAA/CMEPS
```

Comment:
Why are you pointing to a branch of CMEPS?

Reply:
I think it is suggested by Jong to point CMEPS to  [NOAA-EMC/CMEPS@cec8db8](https://github.com/NOAA-EMC/CMEPS/commit/cec8db8d09fa0a0b016d197a68edc67cbd100d97)

Reply:
The issue previously is that you were not using the latest CMEPS hash. If you are not making any changes to CMEPS, you need to use the current develop branch in NOAA-EMC/CMEPS